cli/app.py

```diff
@@ -47,10 +55,15 @@ def audit():
           # print(app.utilities.to_json(data))
           progress.update(100*calls/len(requests))
           summary = check.summarize(evaluated, summary)
+          check.resources += evaluated
```

Comment:
@danjoneslf What's going on here? I can see you have the check.resources list and the evaluated list both set to empty - then you append data to the evaluated list and finally add the evaluated list to the check.resources list? Are both required?

Reply:
It's necessary at the moment because of the clumsy way I've implemented the summarize method. This code is largely a cut and paste from the lambda audit process and I think my thinking was that the evaluate lambda should be broken down into a number of smaller lambda functions so I think I was keeping that inner loop relatively self-contained so you could strip it out write out to SQS and parallelise that bit.